src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.0;netstandard2.0;net461</TargetFrameworks>
```

Comment:
System.Text.Json depends on BCL.AsyncInterfaces but does not target `netstandard2.1`, instead it targets `netcoreapp3.0`.  Making it target `netstandard2.1` requires some significant changes to the implementation for very little benefit, as this TargetFramework would rarely be used as `netcoreapp3.0` would be preferred.

As a result I chose for this and in Logging.EventSource to only eliminate BCL.AsyncInterfaces when targeting `netcoreapp3.0`.  If folks think adding bloat to System.Text.Json is worth it to eliminate the dependency we could revisit that as a separate issue.